test/fake-timers-test.js

```diff
@@ -4655,6 +4655,26 @@ describe("issue #315 - praseInt if delay is not a number", function () {
     });
 });
 
+describe("#368 - timeout.refresh setTimeout arguments", function () {
+    before(function () {
+        if (!addTimerReturnsObject) {
+            this.skip();
+        }
+    });
+    it("should forward  arguments passed to setTimeout", function () {
+        var clock = FakeTimers.install();
+        var stub = sinon.stub();
+
+        var t = setTimeout(stub, 1000, "test");
+        clock.tick(1000);
+        t.refresh();
```

Comment:
Does this pass in browsers?

Reply:
I ran:

```bash
$ npm test
$ npm run test-headless
```

Should I run some other script?

Reply:
That's probably a _separate_ bug (refresh existing when run in browsers), heh.